raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -224,6 +240,13 @@ contract TokenNetwork is Utils {
         }
 
         deposit_limit = 100 * (10 ** uint256(decimals));
+
+        administrator = _administrator;
+    }
+
+    function setSafetyDeprecationSwitch(bool status) onlyAdmin public {
```

Comment:
This looks more like a `toggleSafetyDeprecationSwitch()` function. Seems that it can `toggle` between `true` and `false` values for the switch. If we keep with this behaviour use `toggle` in the name.

But also from what I understood from our conversation we want this to be set only once and to not be settable again. Once it's deprecated that's a final action which should not be undoable. 

Let's discuss this also offline.